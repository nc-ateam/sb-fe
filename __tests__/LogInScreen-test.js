import React from "react";
import "react-native";
import LogInScreen from "../screens/LogInScreen";

import renderer from "react-test-renderer";

describe("<LogInScreen />", () => {
  it("renders without crashing", () => {
    const rendered = renderer.create(<LogInScreen />).toJSON();
    expect(rendered).toBeTruthy();
  });

  it("creates a screenshot", () => {
    const rendered = renderer.create(<LogInScreen />).toJSON();
    expect(rendered).toMatchSnapshot();
  });
});
