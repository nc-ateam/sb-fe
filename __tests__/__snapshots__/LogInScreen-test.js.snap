// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LogInScreen /> creates a screenshot 1`] = `<ExponentAppLoading />`;
